codes/2529_cpp_yji0828.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include <algorithm>
+#include <vector>
+#include <string>
+using namespace std;
+
+int k;
+char sign[11];
+int num[11];
+bool check[11];
+vector<string> answers;
+string maximum, minimum;
+
+bool check_one(char a, char b, char oper);
+void comb(int index, string num);
```

Comment:
comb 내부에서 전역변수인 answer에 가능한 string들을 넣기보다 min,max string을 pair<string,string>형태로 comb가 return하게 짜는 편이 더 깔끔해보입니다. pair를 쓰려면 utility를 include하시면 됩니다.
```suggestion
bool check_one(char a, char b, char oper);
pair<string, string> comb(int index, string num);
```

Reply:
그런데 comb가 재귀적으로 실행되고 있는데 최종 pair를 어떤 식으로 리턴하게 해야하는지 어려운 것 같습니다..ㅜㅜ

Reply:
조금 어려우니 제가 짜드릴게요 코드 분석해보세요.